src/ray/raylet/worker_pool_test.cc

```diff
@@ -97,6 +111,17 @@ TEST_F(WorkerPoolTest, HandleWorkerRegistration) {
   }
 }
 
+TEST_F(WorkerPoolTest, StartupWorkerCount) {
+  int desired_initial_worker_count_per_language = 20;
+  for (int i = 0; i < desired_initial_worker_count_per_language; i++) {
+    worker_pool_.StartWorkerProcess(1000 + i, Language::PYTHON);
+    worker_pool_.StartWorkerProcess(2000 + i, Language::JAVA);
+  }
+  // Check that number of starting worker processes equals to
+  // maximum_startup_concurrency_ * 2. (because we started both python and java workers)
+  ASSERT_EQ(worker_pool_.NumWorkerProcessesStarting(), 5 * 2);
```

Comment:
Can you include a comment saying where the value `5` is initialized otherwise it is a bit out of the blue.

Reply:
done